docs/references/api/resource_embedding.rst

```diff
@@ -1189,6 +1199,39 @@ You can use this to get the columns of a join table in a many-to-many relationsh
      }
    ]
 
+One-to-many and Many-to-many relationships
+------------------------------------------
+
+For these relationships, the spread embedded resource columns will be aggregated into arrays.
+
 .. note::
+  This is disabled by default, since aggregates can cause performance issues.
+  It can be activated by setting :ref:`db-aggregates-enabled` to ``true``.
 
-  The spread operator ``...`` is borrowed from the Javascript `spread syntax <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax>`_.
+.. code-block:: bash
+
+   # curl -g "http://localhost:3000/directors?select=first_name,...films(film_titles:title,film_years:year)&first_name=like.Quentin*"
+
+  curl --get "http://localhost:3000/directors" \
+    -d "select=first_name,...films(film_titles:title,film_years:year)" \
+    -d "first_name=like.Quentin*"
```

Comment:
I've been trying to understand the feature. This example doesn't work on the fixtures we have unfortunately.

So I do:

```bash
curl "localhost:3000/clients?select=*,...projects(id,name)"

{"code":"42803","details":null,"hint":null,"message":"column \"clients.id\" must appear in the GROUP BY clause or be used in an aggregate function"}stevechavez@pop-os:~/Projects/postgrest$
```

And it fails.. Is this expected?

Reply:
Ah, seems it works when specifying the parent columns in select:

```
curl "localhost:3000/clients?select=id,name,...projects(id,name)"

[{"id":1,"name":"Microsoft","id":[1,2],"name":["Windows 7","Windows 10"]}, 
 {"id":2,"name":"Apple","id":[3,4],"name":["IOS","OSX"]}]
``` 

It also works without specifying the parent id, which is good:

```
curl "localhost:3000/clients?select=name,...projects(id,name)"

[{"name":"Microsoft","id":[1,2],"name":["Windows 7","Windows 10"]}, 
 {"name":"Apple","id":[3,4],"name":["IOS","OSX"]}]
```

Is this tested? If not, it would be good to add.

Also, the `*` would have to get expanded to clear this bug. We already have a `expandStars` function, maybe it can be reused.

Reply:
> Ah, seems it works when specifying the parent columns in select:

Yes.

> Is this tested? If not, it would be good to add.

All the tests I added to `SpreadQueriesSpec.hs` for "to-many" spreads use a column that is not the primary key. Which leads me to...

> So I do:
> 
> ```shell
> curl "localhost:3000/clients?select=*,...projects(id,name)"
> 
> {"code":"42803","details":null,"hint":null,"message":"column \"clients.id\" must appear in the GROUP BY clause or be used in an aggregate function"}stevechavez@pop-os:~/Projects/postgrest$
> ```
> 
> And it fails.. Is this expected?

No, not expected. Somehow I missed that use case. I already made use of the `expandStars` function, so I'll have to adapt it further to accept this case.